cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/sync/class-upload-sync.php

```diff
@@ -142,14 +148,8 @@ public function handle_bulk_actions( $location, $action, $post_ids ) {
 		switch ( $action ) {
 			case 'cloudinary-push' :
 				foreach ( $post_ids as $post_id ) {
-					delete_post_meta( $post_id, Sync::META_KEYS['sync_error'] );
```

Comment:
I am wondering if we shouldn't keep these for a while. I suspect that there will be a few `sync_error` keys for a while as we missed some cleanups previously. Perhaps this is not even the best place to proceed with this cleanup. Happy to get your ideas.

Reply:
@pereirinha - The `sync_error` is still stored in the internal meta. This was removing it from standard `post_meta`
It's still kept in `Media::update_post_meta`

Reply:
I also had in mind the syncing and downloading keys, which are stored as plain post metas